backend/frontend/src/screens/BookingInfoScreen.js

```diff
@@ -500,11 +516,15 @@ function BookingInfoScreen() {
                           name: `${slot.start_time}-${slot.end_time}`,
                         }))}
                       />
-                    ) : (
-                      <Alert severity="error">
-                      {t("noSlotsAvailable", {courtName})}                       
-                      </Alert>
-                    )}
+                    ) : nearestSlot ? (
+                        <Alert severity="info">
+                          Currently No slots available in {courtName}, Next slot available on {nearestSlot?.days ? nearestSlot.days : formatDate(nearestSlot.date)} at {nearestSlot?.start_time?.slice(0, 5)}
```

Comment:
This has to be translated.

Reply:
This has been already translated and ready for PR by Tharun